pandas/_testing/asserters.py

```diff
@@ -817,33 +817,33 @@ def assert_extension_array_equal(
 
 # This could be refactored to use the NDFrame.equals method
 def assert_series_equal(
-    left,
-    right,
-    check_dtype=True,
-    check_index_type="equiv",
-    check_series_type=True,
-    check_less_precise=no_default,
-    check_names=True,
-    check_exact=False,
-    check_datetimelike_compat=False,
-    check_categorical=True,
-    check_category_order=True,
-    check_freq=True,
-    check_flags=True,
-    rtol=1.0e-5,
-    atol=1.0e-8,
-    obj="Series",
+    left: Series,
+    right: Series,
+    check_dtype: Union[bool, str] = True,
+    check_index_type: Union[bool, str] = "equiv",
+    check_series_type: bool = True,
+    check_less_precise: Union[bool, int] = no_default,
```

Comment:
@simonjayhawkins how shall we type this?

Reply:
`no_default` is defined here https://github.com/pandas-dev/pandas/blob/master/pandas/_libs/lib.pyx#L2384

Reply:
revealed type is Any, so not yet an issue. Lets defer handling of this till it becomes an issue. i.e. when we merge the type stub from pyright for lib.pyx